SurveyApp/Views/AnswerView/MultipleChoiceAnswerView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  MultipleChoiceAnswerView.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 7/7/21.
+//
+
+import UIKit
+import NimbleExtension
+
+class MultipleChoiceAnswerView: UIView {
+
+    let answers: [SurveyAnswer]
+
+    private let tableView = UITableView()
+
+    init(answers: [SurveyAnswer]) {
+        self.answers = answers
+        super.init(frame: .zero)
+        setUpLayout()
+        setUpViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setUpLayout() {
+        addSubview(tableView)
+        
+        tableView.snp.makeConstraints {
+            $0.leading.equalToSuperview().inset(10.0)
+            $0.trailing.equalToSuperview().inset(20.0)
+            $0.top.equalToSuperview().inset(50.0)
+            $0.height.equalToSuperview().inset(50.0)
```

Comment:
if you want maximum height 300 you can do
```suggestion
            $0.height.lessThanOrEqualTo(300.0)
            $0.height.lessThanOrEqualToSuperview().inset(50.0)
```